packages/Babylonian-UI.package/BPExampleMorph.class/instance/openExampleInDockedPrintbugger.st

```diff
@@ -0,0 +1,5 @@
+private
+openExampleInDockedPrintbugger
+
+	(self containingBrowser sidePane allMorphs detect: [:aSubmorph | 
```

Comment:
Isn't the root morph in the sidePane of kind BPPrintbugger already?

Reply:
no, there is a transformmorph inbetween. showing that access through morph hierarchy is always tricky, so i'd leave it with the search here

Reply:
I see, alright.